### PR TITLE
Amend description of tag usage in the documentation

### PR DESCRIPTION
**Pull request by b10n1k**
The documentation do not make clear what tags exists or whether there are specific ones.
So reading the documentation and wanting to use tagging, user will wonder which tag he needs.
However Openqa does not define further tags other than important in my understanding.

Signed-off-by: ybonatakis <ybonatakis@suse.com>

### CODE REVIEW

File: docs/UsersGuide.asciidoc [hunk 1]
```diff
@@ -364,7 +364,8 @@ Based on comments on the group overview individual builds can be tagged. As
 'build' by themselves do not own any data the job group is used to store this
 information. A tag has a build to link it to a build. It also has a type
 and an optional description. The type can later on be used to distinguish
-tag types.
+tag types. OpenQA does not define further tag types other than _important_ tag.
+User is free to choose any tag he wants for his/her jobs.
```
Comment (Martchus) [inline docs/UsersGuide.asciidoc:368]: The casing of "openQA" is wrong and I'd also tweak the wording a bit, e.g.:
```suggestion
tag types. Note that openQA does not define further tag types besides the _important_ tag.
However, the user is free to choose any tag type as needed.
```

**Review by Martchus — CHANGES_REQUESTED** [review]
In the commit message: `s/However Openqa/However, openQA/`

File: docs/UsersGuide.asciidoc (hunk 1 above)
Comment (b10n1k) [reply docs/UsersGuide.asciidoc]: *this change is ready for review*

File: docs/UsersGuide.asciidoc [hunk 2]
```diff
@@ -364,7 +364,8 @@ Based on comments on the group overview individual builds can be tagged. As
 'build' by themselves do not own any data the job group is used to store this
 information. A tag has a build to link it to a build. It also has a type
 and an optional description. The type can later on be used to distinguish
-tag types.
+tag types.  Note that openQA does not define further tag types besides the _important_ tag.
```
Comment (Martchus) [inline docs/UsersGuide.asciidoc:367]:
```suggestion
tag types. Note that openQA does not define further tag types besides the _important_ tag.
```

Comment (okurz) [reply docs/UsersGuide.asciidoc:367]: and please also wrap lines consistently at 80 characters

**Review by okurz — CHANGES_REQUESTED** [review]

File: docs/UsersGuide.asciidoc (hunk 2 above)
Comment (b10n1k) [reply docs/UsersGuide.asciidoc]: *this change is ready for review*

**Review by okurz — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Pull request merged**